### PR TITLE
Add `futureTargets`

### PR DESCRIPTION
**Pull request by fanatid**
Issue #14

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -86,16 +86,20 @@ var deprecatedTargets = [
...
   {runtime: 'node', target: '2.0.0', abi: '44', lts: false},
   {runtime: 'node', target: '3.0.0', abi: '45', lts: false},
-  {runtime: 'node', target: '8.0.0', abi: '52', lts: false},
   {runtime: 'electron', target: '0.30.0', abi: '44', lts: false},
   {runtime: 'electron', target: '0.31.0', abi: '45', lts: false},
   {runtime: 'electron', target: '0.33.0', abi: '46', lts: false}
 ]
 
-var allTargets = supportedTargets.concat(deprecatedTargets)
+var featureTargets = [
```
Comment (lgeiger) [inline index.js]: `futureTarget` would be clearer, since it isn't released.

Comment (fanatid) [inline index.js]: English is not my native, but should not variable with items (Array) have ending `s`?

Comment (lgeiger) [inline index.js]: You're right, sorry for the typo. I meant `futureTargets` instead of `featureTargets`.

Comment (fanatid) [inline index.js]: Ohhh, sorry, my bad. This is so stupid, I'll fix it!

**Comment by fanatid** [comment]
Fixed. Can not change branch name, sorry ;)

File: index.js (hunk 1 above)
Comment (lgeiger) [inline index.js]: No problem!

**Pull request merged**